docs/sources/tempo/setup/operator/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploy Tempo with Tempo Operator
+menuTitle: Deploy with operator
+description: Learn how to deploy Tempo with Tempo Operator
+weight:
+aliases:
+- /docs/tempo/operator/operator
+---
+
+# Deploy Tempo with Tempo Operator
+
+The Tempo Operator allows you to configure, install, upgrade and operate Grafana Tempo on Kubernetes and OpenShift clusters.
+
+Some of the operator features are:
+
+* **Resource Limits** - an overall resource requests and limits can be specified in the `TempoStack` CR, and the operator will assign fractions of it to each component
+* **AuthN and AuthZ** - OIDC and RBAC support
+* **Multitenancy** - traces of multiple tenants can be stored in the same Tempo cluster
+* **mTLS** - communication between the Tempo components can be secured via mTLS
+* **Jaeger UI** - traces can be visualized in Jaeger UI exposed via Ingress or OpenShift Route
+* **Observability** - the operator and `TempoStack` operands expose telemetry (metrics, traces) and integration with Prometheus `ServiceMonitors`
```

Comment:
Under multitenancy, "Traces of multiple tenants" soundsl ike there are remnants of tenants that can be stored. Should that be traces *from* multiple tenants? Or that Multiple tenants can send traces? Feel free to edit this comment to update as you think best. 
```suggestion
* **Resource Limits** - Specify overall resource requests and limits in the `TempoStack` CR; the operator assigns fractions of it to each component
* **AuthN and AuthZ** - Supports OpenID Control (OIDC) and role-based access control (RBAC)
* **Multitenancy** - The same Tempo cluster can store Traces from multiple tenants * **mTLS** - Communication between the Tempo components can be secured via mTLS
* **Jaeger UI** - Traces can be visualized in Jaeger UI exposed via Ingress or OpenShift Route
* **Observability** - The operator and `TempoStack` operands expose telemetry (metrics, traces) and integration with Prometheus `ServiceMonitors`
```

Reply:
I have changed it to `Multiple tenants can send traces to the same Tempo cluster`.